### PR TITLE
[react-native]: Remove `Navigator` and `NavigationExperimental`

### PR DESCRIPTION
**Pull request by newyankeecodeshop**
They were both deprecated in React Native 0.44 and no longer available in current builds.

Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

Select one of these and delete the others:

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: https://github.com/facebook/react-native/commit/92d985fb4982e739f66582ae0cc08c7312a6ef4a

### CODE REVIEW

File: types/react-native/index.d.ts [hunk 1]
```diff
@@ -4831,87 +4760,9 @@ export interface Route {
     index?: number
     onRightButtonPress?: () => void
     rightButtonTitle?: string
-    sceneConfig?: SceneConfig
     wrapperStyle?: any
```
Comment (newyankeecodeshop) [inline types/react-native/index.d.ts]: The `Route` interface is still used by `NavigatorIOS`, which is not deprecated.

File: types/react-native/index.d.ts [hunk 2]
```diff
@@ -8496,303 +8144,6 @@ export interface ImageStoreStatic {
         ): void
 }
 
-export interface TabsReducerStatic {
-    JumpToAction(index: number): any;
-}
-
-export type TabsReducerFunction = (params: any) => any;
```
Comment (newyankeecodeshop) [inline types/react-native/index.d.ts]: I'm not sure where `TabsReducerStatic` came from, but it's not referenced anywhere.

**Comment by alloy** [comment]
Good riddance 👍

I’ll merge when @kaoDev gives his thumbs-up as well.

(Looks like you’ll need to rebase btw.)

**Comment by kaoDev** [comment]
the change in reat-native-tab-view looks fine to me, so here is my 👍

**Comment by alloy** [comment]
@andy-ms Looks like there’s still a few cases left that need updating.

@newyankeecodeshop Your PR didn’t introduce those, so gonna merge now regardless of red CI 👍

**Comment by newyankeecodeshop** [comment]
Thanks @alloy I was wondering how that could have broken.

**Pull request merged**